File: app/src/main/java/com/github/doomsdayrs/apps/shosetsu/ui/reader/demarkActions/IndentChange.java
package com.github.doomsdayrs.apps.shosetsu.ui.reader.demarkActions;

import com.github.doomsdayrs.apps.shosetsu.backend.Utilities;
import com.github.doomsdayrs.apps.shosetsu.ui.reader.NewChapterReader;

import static com.github.doomsdayrs.apps.shosetsu.backend.Utilities.changeIndentSize;

public class IndentChange implements Utilities.DemarkAction {
    private final NewChapterReader chapterReader;

    public IndentChange(NewChapterReader chapterReader) {
        this.chapterReader = chapterReader;
    }

    @Override
    public void action(int spared) {
        changeIndentSize(spared);
        chapterReader.currentView.setUpReader();
    }
}

File: app/src/main/java/com/github/doomsdayrs/apps/shosetsu/ui/reader/demarkActions/IndentChange.kt
package com.github.doomsdayrs.apps.shosetsu.ui.reader.demarkActions

import com.github.doomsdayrs.apps.shosetsu.backend.DeMarkAction
import com.github.doomsdayrs.apps.shosetsu.common.Settings
import com.github.doomsdayrs.apps.shosetsu.ui.reader.ChapterReader

class IndentChange(private val chapterReader: ChapterReader) : DeMarkAction {
	override fun action(spared: Int) {
		Settings.ReaderIndentSize = spared
		//chapterReader.setUpReader()
	}
}